packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/TerariumAsset.java

```diff
@@ -66,11 +66,20 @@ protected void onUpdate() {
 	@TSOptional
 	private Boolean publicAsset = false;
 
+	// This is here just to satisfy the service interface.
 	public TerariumAsset clone() {
-		try {
-			return (TerariumAsset) super.clone();
-		} catch (final CloneNotSupportedException e) {
-			throw new RuntimeException("Error cloning TerariumAsset", e);
-		}
+		throw new RuntimeException(
+				"This should not be called. Use explicitly override this method on the derived class and call cloneSuperFields instead.");
```

Comment:
Is that English?
Use explicitly override

Reply:
lol, hold on lemme fix.